actions/snapshots.go

```diff
@@ -12,6 +12,74 @@ import (
 	"github.com/gobuffalo/buffalo"
 )
 
+// SnapshotsPost creates a manual snapshot for a given database instance or cluster
+func SnapshotsPost(c buffalo.Context) error {
+	req := SnapshotCreateRequest{}
+	if err := c.Bind(&req); err != nil {
+		log.Println(err)
+		return c.Error(400, err)
+	}
+
+	if req.SnapshotIdentifier == nil {
```

Comment:
you also want the string to be not empty i think.  i go back and forth on using pointers of concrete types in request/response structs.  its useful in a lot of places to just check `!= nil`, but this is an example where more is required.